docs/product-administration-and-configuration-guide/src/main/asciidoc/kie-server-creating-containers-proc.adoc

```diff
@@ -0,0 +1,61 @@
+[id='kie-server-creating-containers-proc']
+= Creating Containers
+
+Containers are self-contained environments that have been provisioned to hold instances of your packaged and deployed rule instances.
+
+.Prerequisite
+Once the {KIE_SERVER} is registered, you can start adding containers.
+
+.Procedure
+. Log in to {CENTRAL}.
+. In the main menu, click *Menu* -> *DevOps* -> *Execution Servers*.
+. Click *SERVER TEMPLATES* -> *+ New Server Template*.
+. In the window, click *Container*.
+. Enter the *Group Name*, *Artifact Id*, and *Version* manually.
++ 
+Alternatively, enter the name of your container and search for the project you want to deploy in the container.
+
+. Click *Select* next to the project to automatically enter the details of the project.
+
++
+[WARNING]
+====
+When entering the container's version number, do _not_ use the `LATEST` or `RELEASE` keywords. This feature has been deprecated and can cause deployment issues.
+====
+. If you want to deploy the container automatically, select the *Start Container?* box.
```

Comment:
To deploy the container automatically, select the ....

Reply:
done